starter_code/css/tablet.css

```diff
@@ -0,0 +1,350 @@
+@media (min-width: 768px){
+	/*tablet portrait*/
+
+.container{
+	max-width: 760px;
+}
+
+.logo {
+  font-size: 50px;
+  font-weight: 800;
+}
+
+header {
+  padding: 25px 0;
+}
+
+header .logo {
+  color: #fff;
+  float: left;
+}
+
+nav {
+  float: right;
+  padding-top: 25px;
+  padding-left: 20px;
+}
+
+nav ul{
+
+}
+
+header nav ul li {
+  float: left;
+  margin-top: 15px;
+  margin: 0 0 0 40px;
+  font-size: 40px;
```

Comment:
Same here. Don't repeat styles if they are defined already on smaller screens. The smaller screen styles still affect larger devices. So you should really only add styles for larger devices that need to change what was already applied to smaller devices